dev-requirements.txt

```diff
@@ -1,6 +1,7 @@
 pylint==3.0.2
 httpretty==1.1.4
 mypy==0.931
```

Comment:
This can be removed. I didn't because I don't know who is using it...

Reply:
I think there's no usage on this repo. But we are using mypy in core + a basic setup of pyright with typeCheckingMode off.